xray/backends/netCDF4_.py

```diff
@@ -54,7 +54,7 @@ def _nc4_values_and_dtype(var):
     return var, dtype
 
 
-def _nc4_group(ds, group):
+def _nc4_group(ds, group, mode):
```

Comment:
At some point, it could be nice to have a full API for loading a set of datasets from groups. But not in this PR.